scipy/stats/_continuous_distns.py

```diff
@@ -4076,6 +4076,101 @@ def fit(self, data, *args, **kwds):
 laplace = laplace_gen(name='laplace')
 
 
+class laplace_asymmetric_gen(rv_continuous):
+    r"""An asymmetric Laplace continuous random variable.
+
+    %(before_notes)s
+
+    Notes
+    -----
+    The probability density function for `laplace_asymmetric` is
+
+    .. math::
+
+        f(x, \kappa) &= \frac{1}{\kappa+\kappa^{-1}}\exp(-x\kappa),\quad x\ge0\\
+                     &= \frac{1}{\kappa+\kappa^{-1}}\exp(x/\kappa),\quad x<0\\
+
+    for :math:`-\infty < x < \infty`, :math:`\kappa > 0`.
+
+    `laplace_asymmetric` takes ``kappa`` as a shape parameter for
+    :math:`\kappa`. For :math:`\kappa = 1`, it is identical to a
+    Laplace distribution.
+
+    %(after_notes)s
+
+    See Also
+    --------
+    laplace : Laplace distribution
+
+    References
+    ----------
+    .. [1] Kozubowski TJ and Podg\'orski K. A Multivariate and
+           Asymmetric Generalization of Laplace Distribution,
+           Computational Statistics 15, 531--540 (2000).
+           :doi:`10.1007/PL00022717`
+
+    %(example)s
+
+    """
+    def _rvs(self, kappa, size=None, random_state=None):
```

Comment:
I checked and this does look equivalent to running the uniform random numbers through the `_ppf`. In that case, 

- If we can handle the overhead, why not just use the default rvs implementation?
- If we can't handle the overhead, why are the implementations of these functions different? Is there a property of uniformly distributed random numbers that makes this implementation tend to be more efficient than the implementation in `_ppf`?

Reply:
I'm not familiar enough with `stats` internals to comment. and defer to you all.

Reply:
This mod to `_rvs` deleted per discussion below.